### PR TITLE
Fixed inplace editing bug and wrong script name

### PR DESCRIPTION
**Pull request by eyck**
Sorry, but after installing via pip I encountered a bug wrt. to the --inplace option and a flaw in the naming of generated scripts

### CODE REVIEW

File: gcodeutils/gcode_optimize_arcs.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def main():
     GCodeArcOptimizerFilter().filter(gcode)
 
     # write back modified gcode
-    gcode.write(args.outfile if args.inplace is True else args.outfile)
+    gcode.write(open(args.infile.name, 'w') if args.inplace is True else args.outfile)
```
Comment (zeograd) [inline gcodeutils/gcode_optimize_arcs.py:47]: isn't `gcode.write(args.infile.name if args.inplace else args.outfile)` enough to have inplace editing working ?

Comment (eyck) [inline gcodeutils/gcode_optimize_arcs.py:47]: Nope, unfortunately not as args.infile is a file object opened for read.

Comment (zeograd) [inline gcodeutils/gcode_optimize_arcs.py:47]: Right. it just won't make sense with both --inplace and the default of sys.stdin, but people looking for stdin/stdout piping don't need --inplace. I'm fine not guard checking it.

**Comment by eyck** [comment]
Will get back to you when completely done. Have issues in one or the other real world examples

**Comment by zeograd** [comment]
Ok, take the time you need.

**Pull request closed without merging**